DOCS/man/options.rst

```diff
@@ -4442,6 +4443,33 @@ Screenshot
     Set the JPEG XL compression effort. Higher effort (usually) means better
     compression, but takes more CPU time. The default is 4.
 
+``--screenshot-avif-encoder=<encoder>``
+    Specify the AV1 encoder to be used by libavcodec for encoding avif
+    screenshots.
+
+    Default: ``libaom-av1``
+
+``--screenshot-avif-pixfmt=<format>``
+    Specify the pixel format to the libavcodec encoder.
```

Comment:
This doesn't make much sense for an user choice.
Because of this mpv won't:
1. automatically pick the right subsampling (420, 422, 444) depending on the input data
2. automatically pick a high bit depth format depending on the input data

it could additionally cause problems with `image_writer_high_depth` since the user's chosen pixfmt may be low-depth

Reply:
An equivalent option to `screenshot-jpeg-source-chroma` would be appropriate if that is the case.
But leaving yuv420p as a default with no autodetection is definitely bad.